cohort-translation/src/test/resources/model-info.xml

```diff
@@ -0,0 +1,624 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  ~ /*
+  ~  * (C) Copyright IBM Corp. 2021
+  ~  *
+  ~  * SPDX-License-Identifier: Apache-2.0
+  ~  */
+  -->
+
+<modelInfo xmlns="urn:hl7-org:elm-modelinfo:r1" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:xsd="http://www.w3.org/2001/XMLSchema" name="HI" version="0.0.1" url="http://mock.hi.model" targetQualifier="IDontKnow" patientClassName="DrugClaim" patientBirthDatePropertyName="none">
+    <typeInfo namespace="HI" name="DrugClaim" identifier="http://mock.hi.model/DrugClaim" label="DrugClaim" retrievable="true" xsi:type="ClassInfo">
```

Comment:
Given this is HI-based, are we allowed to have it in the public repo? Might be a question for a morning meeting/Slack.

Reply:
It's a fair question, I think you're right. For now I'm trading it out for an already committed model info.